scripts/reset

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
```

Comment:
'reset' doesn't quite describe what the script does, as it also runs the application

Reply:
Fair point. Perhaps it shouldn't run the application?

Reply:
I'd prefer a single command to start the application with a clean state rather than have to run a combination of commands. Renaming this command would be better.

Reply:
Refactored `build`, `start`, `reset` and `connected-start` into a single `start` script with arguments to cover all the different uses of those scripts.